### PR TITLE
[BEAM-261] Make translators package private.

### PR DESCRIPTION
**Pull request by tweise**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:
- [x] Make sure the PR title is formatted like:
  `[BEAM-<Jira issue #>] Description of pull request`
- [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
     Travis-CI on your fork and ensure the whole test matrix passes).
- [x] Replace `<Jira issue #>` in the title with the actual Jira issue
     number, if there is one.
- [ ] If this contribution is large, please file an Apache
     [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).
---

R: @kennknowles 
Leftover from original PR.

### CODE REVIEW

**Comment by kennknowles** [comment]
Incidentally, the Travis config on `master` has been improved a lot so if you merge you may get a cleaner signal, with just the Eclipse build failing.

File: runners/apex/src/main/java/org/apache/beam/runners/apex/ApexRunner.java [hunk 1]
```diff
@@ -28,7 +28,8 @@
 import java.util.Arrays;
 import java.util.List;
 
-import org.apache.beam.runners.apex.translators.TranslationContext;
+import org.apache.beam.runners.apex.translation.ApexPipelineTranslator;
+import org.apache.beam.runners.apex.translation.TranslationContext;
```
Comment (kennknowles) [inline runners/apex/src/main/java/org/apache/beam/runners/apex/ApexRunner.java]: This import makes me think that maybe `ApexTranslator.translate(Pipeline, DAG)` would make sense, to make it so this class does not need to depend on the `TranslationContext`. It seems like an implementation detail leak.

File: runners/apex/src/main/java/org/apache/beam/runners/apex/translation/FlattenPCollectionTranslator.java [hunk 2]
```diff
@@ -16,17 +16,17 @@
...
 
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 
-import org.apache.beam.runners.apex.translators.functions.ApexFlattenOperator;
-import org.apache.beam.runners.apex.translators.io.ApexReadUnboundedInputOperator;
-import org.apache.beam.runners.apex.translators.io.ValuesSource;
+import org.apache.beam.runners.apex.translation.functions.ApexFlattenOperator;
+import org.apache.beam.runners.apex.translation.io.ApexReadUnboundedInputOperator;
```
Comment (kennknowles) [inline runners/apex/src/main/java/org/apache/beam/runners/apex/translation/FlattenPCollectionTranslator.java]: Do the `functions` and `io` package add anything? It seems like it forces some things to be public that don't need to be. Or maybe if someone really declares a dependency of `beam-runners-apex` then having access to those would be useful? I almost think all the classes could reside in one package. Users mostly just need the `ApexRunner` and it isn't that many classes. Just vague questions; I trust your understanding and judgment.

**Comment by tweise** [comment]
@kennknowles addressed comments

**Pull request merged**